packages/NodeNameResolver/NodeNameResolver.php

```diff
@@ -181,6 +186,12 @@ public function isStringName(string $resolvedName, string $desiredName): bool
             return $desiredName === $resolvedName;
         }
 
+        foreach(self::REGEX_WILDCARD_CHARS as $char) {
+            if (str_contains($desiredName, $char)) {
+                throw new ShouldNotHappenException('Matching of regular expressions is no longer supported');
```

Comment:
Thanks :+1: 

I'd add most common replacement solution like:
 "Use exact compare of $this->getName() and str_ends_with() or str_starts_with() instead"

Reply:
good point, fixed